oncrpc4j-core/src/main/java/org/dcache/oncrpc4j/rpc/RpcAuthTypeUnix.java

```diff
@@ -70,7 +73,18 @@ public void xdrDecode(XdrDecodingStream xdr) throws OncRpcException, IOException
         _gids = xdr.xdrDecodeIntVector();
         _verifier.xdrDecode(xdr);
 
-        _subject = Subjects.of(_uid, _gid, _gids);
+        _subject = buildUnixSubject(_uid, _gid, _gids);
+    }
+
+    private Subject buildUnixSubject(int uid, int gid, int[] gids) {
```

Comment:
It seems that this can be made static

Reply:
Sure, I can update the patch with that change. It may be a day or so before I get to it.

Reply:
force-pushed ec7ce13 with suggested change from @rumpelstiltzkin